### DISCUSSION TITLE
Where are new versions of PF published?

### DISCUSSION THREAD

**Discussion by wlhUser** [primefaces/community#227] [PrimeFaces] [answered]
With version 13 of PrimeFaces it is quite unclear for me where the new versions are published.
On the 'official' download page, https://www.primefaces.org/downloads/ the version 13 is not even mentioned.
On the PrimeFaces ELITE page (where I used to get the new versions) version 13 is also not mentioned.

Instead I can find the new versions in the maven repo.

Please think about adding links to the newest versions at least in the ELITE section.

**Comment by wnibes** [comment]
For the moment give this a try
https://mvnrepository.com/artifact/org.primefaces/primefaces
13.0.0, 13.0.1, 13.0.2 are there

**Reply by melloware** [reply, marked as answer]
@wlhUser PrimeFaces for 13 has switched its model and new community driven releases 13.0.1, 13.0.2 etc will be in Maven Central.  If you are using an older PrimeFaces version they are going to offer LTS Subscriber support.    The concept of "Elite" is officially no more.

**Reply by Thomas-Schindler** [reply]
HI @melloware ,

does that mean, that if our company has subscribed for PrimeFaces Elite versions and we use only PrimeFaces Version >= 13.0.0, we don't need to refresh our Elite subscription anymore?
Kind Regards
Thomas

**Reply by melloware** [reply]
I am not positive but yes I think the Elite subscription is going away.

**Comment by cagataycivici** [comment]
Elite will be replaced by PrimeFaces LTS by the end of this year and will be announce in early January after holidays. Same model as in PrimeNG LTS.

http://primeng.org/lts

**Comment by MikeMV97** [comment]
I have a doubt about older versions, I want to get primefaces 8.0.25. In my case I still require a Elite subscription to download it?

**Reply by melloware** [reply]
Your PrimeStore account with your Elite releases still exists.

**Reply by MikeMV97** [reply]
Excuse me for not clarifying it. I don't have a subscription yet, do I need to purchase it?

**Reply by melloware** [reply]
If you need access to 8.0.25 then yes you would need an Elite subscription.
